.github/workflows/ci-test.yml

```diff
@@ -1,17 +1,33 @@
-name: CI
+name: Automated tests
 
 on:
   push:
     branches: [ main ]
+    paths:
+        - 'package.json'
+        - 'src/**/*.html'
+        - 'src/**/*.js'
+        - 'src/**/*.json'
+        - 'tests/**/*.html'
+        - 'tests/**/*.js'
+        - 'tests/**/*.json'
   pull_request:
     branches: [ main ]
+    paths:
+        - 'package.json'
+        - 'src/**/*.html'
+        - 'src/**/*.js'
+        - 'src/**/*.json'
+        - 'tests/**/*.html'
+        - 'tests/**/*.js'
+        - 'tests/**/*.json'
```

Comment:
This duplication is unfortunate and error prone. Can you see if there is a way around it, or if it is just a limitation of GitHub Actions at the moment?

Reply:
It's a limitation at the moment.